tests/bots.py

```diff
@@ -47,6 +47,8 @@
 
 FALLBACKS = json.loads(base64.b64decode(FALLBACKS).decode("utf-8"))  # type: list[dict[str, str]]
 
+chosen_bot = {}
```

Comment:
please upper-case this

Reply:
I'll make add it in a class so I guess I won't have to upper case